src/telegram_bot.py

```diff
@@ -0,0 +1,55 @@
+from os import getenv
+from telegram.ext import Updater, CommandHandler, CallbackContext
+import logging
+import schedule
+from requests import get
+from story import Story, get_new_stories
+
+# enables and get logger
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+					 level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+# fetching env variables
+TOKEN = getenv("TELEGRAM_TOKEN")
+JSON_URL = getenv("JSON_URL")
+FETCH_INTERVAL = int(getenv("FETCH_INTERVAL", default=900)) # in seconds
```

Comment:
Dato che nel `README` è specificato che `FETCH_INTERVAL` è in minuti, moltiplicherei questo numero per 60.